.changeset/cool-maps-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+"@salt-ds/window": patch
```

Comment:
Given this is the first merge to `main`, can we clean / merge the changesets so it makes more sense when it appears in changelog.